site/content/usage/06-vpc-networking.md

```diff
@@ -143,3 +143,20 @@ nodeGroups:
     kubeletExtraConfig:
         clusterDNS: ["169.254.20.10","172.20.0.10"] 
 ```
+
+### NAT Gateway
+
+The NAT Gateway for a cluster can be configured through config files. The available options are: disabled, single 
```

Comment:
Can we follow the same casing as in the yaml? `Disabled, Single`

Reply:
I think we should also mention that this is configurable via a CLI arg as well: `--vpc-nat-mode`.